diff --git a/src/watchdog/__init__.py b/src/watchdog/__init__.py
index 546f40f..2e03622 100755
--- a/src/watchdog/__init__.py
+++ b/src/watchdog/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 #
 # Copyright 2017-2018 Amazon.com, Inc. and its affiliates. All Rights Reserved.
 #
@@ -280,7 +280,7 @@ def assert_root():
 
 
 def read_config(config_file=CONFIG_FILE):
-    p = ConfigParser.SafeConfigParser()
+    p = ConfigParser()
     p.read(config_file)
     return p
 
